eprihlaska/forms.py

```diff
@@ -76,6 +76,8 @@ class MoreDetailPersonalDataForm(FlaskForm):
 
     place_of_birth = SelectField(label=c.BIRTH_PLACE,
                                  choices=choices_from_csv(DIR + '/data/obce.csv',
+                                                          ['id', 'Názov obce'],
```

Comment:
@JakubNvk Neviem presne, ako sa to sem dostalo, ale toto sem dva krat netreba. Malo by to tiez riesit problem s `TypeError: choices_from_csv() got multiple values for argument 'fmt'`

Reply:
Uprimne netusim, spadol mi pocas prace terminal - je dost mozne ze to narobilo zvlastny bordel..